inscriptions/ISic003659.xml

```diff
@@ -191,76 +196,32 @@
-                    </expan>
-                    <lb n="2"/>Feridius Cereris dominae
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>s</abbr>
-                            <ex>ervus</ex>
-                        </expan>
-                    </supplied>
-                    <lb n="3"/>hic sibi
-                    <choice>
-                        <sic>suabiter</sic>
-                        <corr>suaviter</corr>
-                    </choice>
-                    <expan>
-                        <abbr>fec</abbr>
-                        <ex>it</ex>
-                    </expan>
-                    e
-                    <supplied reason="lost">o</supplied>
-                    <lb n="4" break="no"/>rum tres adulescentes<lb n="5"/>quorum nomina lege<lb n="6"/>Onesimus et
-                    <expan>
-                        <abbr>L</abbr>
-                        <ex>ucius</ex>
-                    </expan>
-                    Valerius<lb n="7"/>Cas
-                    <expan>
-                        <ex>s</ex>
-                        <abbr>ianus</abbr>
-                    </expan>
-                    et Filumenus<lb n="8"/>unus cum
-                    <choice>
-                        <sic>mulierae</sic>
-                        <corr>muliere</corr>
-                    </choice>
-                    ea<lb n="9"/>Taurus multis annis
-                    <choice>
-                        <sic>habe</sic>
-                        <corr>have</corr>
-                    </choice>
-                    <expan>
-                        <abbr>facian</abbr>
-                        <ex>t</ex>
-                    </expan>
-                    <lb n="10"/>co
-                    <expan>
-                        <ex>n</ex>
-                        <abbr>iu</abbr>
-                        <ex>n</ex>
-                        <abbr>ximus</abbr>
-                    </expan>
-                    <lb n="11"/>propitiam.
+                    <lb n="1"/><num value="17">XVII</num> <expan><abbr>K</abbr><ex>alendas</ex></expan>
+                    <expan><abbr>Septemb</abbr><ex>res</ex></expan>
+                    <lb n="2"/><persName><name>Feridius</name></persName> <name>Cereris</name> dominae
+                    <supplied reason="lost"><expan><abbr>s</abbr><ex>ervus</ex></expan></supplied>
+                    <lb n="3"/>hic sibi <choice><sic>suabiter</sic><corr>suaviter</corr></choice>
+                    <expan><abbr>fec</abbr><ex>it</ex></expan> e<supplied reason="lost">o</supplied>
+                    <lb n="4" break="no"/>rum tres adulescentes
+                    <lb n="5"/>quorum nomina lege
+                    <lb n="6"/><persName><name>Onesimus</name></persName> et <persName><name><expan><abbr>L</abbr><ex>ucius</ex></expan></name> <name>Valerius</name>
+                    <lb n="7"/><name>Cas<supplied reason="omitted">s</supplied>ianus</name></persName>
```

Comment:
here too, I would prefer to use choice with orig and reg for Casianus versus Cassianus, rather htan supplied reason="omitted" [we're not correcting ancient writers, rather noting their spelling variants]